arcaflow_plugin_pcp/pcp_schema.py

```diff
@@ -54,24 +85,22 @@ class PcpInputParams:
             + validation_warning
         ),
     ] = None
-    generate_csv: typing.Annotated[
-        typing.Optional[bool],
-        schema.name("generate CSV output"),
-        schema.description(
-            "Generates the data payload also in CSV format. This output goes to "
-            "the debug_logs, or to stderr if the --debug flag is used."
-        ),
-    ] = False
-    flatten: typing.Annotated[
-        typing.Optional[bool],
-        schema.name("flatten JSON structure"),
+
+
+@dataclass
+class PostProcessParams(PcpGlobalParams):
+    archive_path: typing.Annotated[
+        str,
+        validation.pattern(file_path_pattern),
+        schema.name("archive file path"),
         schema.description(
-            "Processes the metrics first into a two-dimensional format via the "
-            "pcp2csv converter, and then converts the CSV to JSON, effectively "
-            "flattening the data structure. This is useful when indexing metrics "
-            "to a service like Elasticsearch."
+            "The file system path to the PCP archive file. The path should include the "
+            "name of the archive without a file extension."
         ),
-    ] = False
+    ] = "."
+
+
+post_process_params_schema = plugin.build_object_schema(PostProcessParams)
```

Comment:
Is this definition used?  (If so, why is there no similar definition which references `PcpInputParams`?)

Reply:
Yes, it is imported and used in the plugin, and it is only needed for this class.

Reply:
Incidentally, the similar line for `perf_output_schema` is in fact not used, so I'll remove that one.